src/Akeneo/Tool/Component/Batch/Step/ItemStep.php

```diff
@@ -94,6 +94,11 @@ public function getWriter()
         return $this->writer;
     }
 
+    public function isTrackable(): bool
+    {
+        return true; // @TODO validate reader is trackable
```

Comment:
Something like this ?
```suggestion
        return $this->reader instanceof TrackableReaderInterface;
```

Reply:
seen IRL